Libraries/LibJS/Runtime/Function.h

```diff
@@ -66,12 +66,12 @@ class Function : public Object {
 
 protected:
     explicit Function(Object& prototype);
-    Function(Object& prototype, Value bound_this, Vector<Value> bound_arguments);
+    Function(Object& prototype, Value bound_this, MarkedValueList bound_arguments);
 
 private:
     virtual bool is_function() const override { return true; }
     Value m_bound_this;
-    Vector<Value> m_bound_arguments;
+    MarkedValueList m_bound_arguments;
```

Comment:
This is also already marked by `Function::visit_children()`.

Reply:
Hmmm...at this point, I question: what _isn't_ marked and is stored in a `Vector<Value>`?

The commit message certainly no longer applies.

Reply:
Hmm. The stuff returned from `IndexedProperties::values_unordered()` isn't marked.

Reply:
`SimpleIndexedPropertyStorage::elements()` returns a vector by copy (for no reason I think), which won't be marked

Reply:
All the `ValueAndAttributes` usage looks really suspicious and unmarked actually

Reply:
Aside from the odd copies, it looks like they _are_ marked, through `Object::visit_children()`, `values_unordered()`.

Reply:
It's a bit of a maze to go through, but I don't think your forgotten objects are here...